### PR TITLE
Issue 317 of front end fixed

### PR DESCRIPTION
**Pull request by Gauravseta**
Please deploy this in backend with code whose PR is submitted at front end.

### CODE REVIEW

**Comment by huangqun** [comment]
@Gauravseta your commit has a LOT of coding style / indention issues, these must be fixed before we can consider accepting your fix.

Also, whenever you update the database, you should provide a script to alter existing database so we don't loose data.

**Comment by Gauravseta** [comment]
Alter db table script is not provided.I just added my column in table itself.

**Comment by huangqun** [comment]
We need a script or statement to alter the database, you need to provide that. Thanks!

**Comment by Gauravseta** [comment]
So I cannot add directly inside db table rite?
under SQLs folder I need to provide one more .SQL file which mentions alter table statement rite?
Request you to clarify on this I am confused.

**Comment by huangqun** [comment]
The alter statement / script is in addition to the changes you've made to the code, basically we need BOTH.

You don't need to provide one more .sql file, you can just paste the alter statement in your new PR.

**Comment by Gauravseta** [comment]
OK I will not add new .SQL file.
So I have to remove locale_I'd column from mentor and mentee table and add alter table statement in init.SQL rite? 
Please clarify because people like chq-matteo unecesarily want to steal the issues just because PR is rejected due to minor issue.

**Comment by huangqun** [comment]
No, you should keep that column for the tables, **and** provide the alter statement here.

Please make sure you do everything correctly this time, there are just too many issues with the code you submitted and that doesn't meet our standards. We need all code to be properly formatted (consistent with our existing coding style) and documented, and of course it must be fully working!

**Comment by Gauravseta** [comment]
It means that if PR is rejected twice so another person is eligible to steal issues from actual person, and as code might be already seen by the all so the new person will just modify minor issues such as indentation and it will get accepted so deserving person lost it.That's a good rule.

**Comment by Gauravseta** [comment]
Issue_331 may have indentation and documentation issues but this issue_317 is correct and m still confused what exactly is required for issue_317. so it all sums up to that rejecting PR makes everyone else eligible to steal issues.

Shall I write alter statement in comment box?

File: src/main/java/com/livingprogress/mentorme/services/springdata/MenteeServiceImpl.java [hunk 1]
```diff
@@ -158,5 +163,13 @@ public boolean confirmParentConsent(String token) throws MentorMeException {
     public InstitutionAffiliationCode findInstitutionAffiliationCode(String code) throws MentorMeException {
         return institutionAffiliationCodeRepository.findByCode(code);
     }
+    
+    
+
+	@Override
+	public Locale findLocale(String value) throws MentorMeException {
+		// TODO Auto-generated method stub
```
Comment (huangqun) [inline src/main/java/com/livingprogress/mentorme/services/springdata/MenteeServiceImpl.java]: code is not documented either.

File: src/main/java/com/livingprogress/mentorme/services/springdata/MentorServiceImpl.java [hunk 2]
```diff
@@ -29,6 +30,10 @@
     @Autowired
     private MenteeMentorProgramRepository menteeMentorProgramRepository;
 
+    @Autowired
+    private LocaleRepository localeRepository;
```
Comment (huangqun) [inline src/main/java/com/livingprogress/mentorme/services/springdata/MentorServiceImpl.java]: missing documentation

File: src/main/java/com/livingprogress/mentorme/services/springdata/MentorServiceImpl.java [hunk 3]
```diff
@@ -128,5 +133,13 @@ public Mentor update(long id, Mentor entity) throws MentorMeException {
         }
         return existing;
     }
+    
+    
+
+	@Override
+	public Locale findLocale(String value) throws MentorMeException {
```
Comment (huangqun) [inline src/main/java/com/livingprogress/mentorme/services/springdata/MentorServiceImpl.java]: No documentation.

**Comment by huangqun** [comment]
Did you see my comments above? There were a lot of code indention issues and some useless comments too. And also, carefully check that every new code / method / variable is properly documented, there might be issues that I still missed.

So you need to double check that everything is correct and complete before submitting the next PR.
And yes you can put the alter statement in the comment of the new PR.

There's actually not a fixed number of trials a member can have to fix his/her PR based on feedback, but if it takes too many iterations we'll open it up to other members. Or if the fix is terribly wrong we'll also open it for other members. I hope this explains it.

**Comment by Gauravseta** [comment]
Surely i will fix everything in next PR.

**Comment by huangqun** [comment]
Thanks!

**Comment by huangqun** [comment]
@Gauravseta forgot to mention, you have to make sure that the unit tests don't fail after making the changes to backend.

**Comment by Gauravseta** [comment]
@huangqun I will make sure everything is okay now with proper indentation, proper commenting either in comment box or in code itself. All my code Is working  properly I tested well..all I need to resolve minor issues.I will cover everything now

**Comment by Gauravseta** [comment]
Keeping apart indentation and commenting issues, request you to tell did functionality work for this issue as I need locale_I'd column in mentor and mentee table for other issues solving too.based on this issue  my other issue is dependent.

**Pull request closed without merging**